src/components/Form/From/NumericInput/helpers.ts

```diff
@@ -1,3 +1,4 @@
+import { ChangeEvent, ClipboardEvent, KeyboardEvent } from 'react';
```

Comment:
I don't think that we use `ClipboardEvent` in this file.

Reply:
It's used in l. 56.